recipes/_install-windows.rb

```diff
@@ -35,6 +35,8 @@
 # Install the package
 windows_package 'Datadog Agent' do
   source temp_file
-  options %(APIKEY="#{node['datadog']['api_key']}" HOSTNAME="#{node['hostname']}" TAGS="#{node['tags'].join(',')}")
+  installer_type :msi
+  options '/norestart ALLUSERS=1'
```

Comment:
Makes sure that the service does not restart and that it is installed globally for all users.  This will cause a bunch of issues down the road since the dd-agent msi does not install on a perMachine bases as illustrated by the following issue https://github.com/DataDog/dd-agent/issues/1689